api/server/router/image/image_routes.go

```diff
@@ -129,7 +129,7 @@ func (ir *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrit
 
 			resp, err := remotecontext.GetWithStatusError(u.String())
 			if err != nil {
-				return err
+				return errdefs.InvalidParameter(err)
```

Comment:
I think `GetWithStatusError` above already is used to set the right status; we should check if that's always the error-type we want to return, but we probably shouldn't always consider this an invalid parameter https://github.com/moby/moby/blob/980d8e5d54d8df31ba0e75e1d9dd474d9d91308a/builder/remotecontext/remote.go#L46-L78